### PR TITLE
Add JDL Parser and Queries

### PR DESCRIPTION
**Pull request by STACKL0RD**
* JDL Parser code + tests
* Configuation DB
* Queries + tests

### CODE REVIEW

File: Parser/jdl_parser.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import sys
+import json
+import os
+
+sys.path.append(sys.path[0].replace("/Parser", ""))
+
+from utils.jdl import nameCluster
+#from utils.jdl import jdlKeys
```
Comment (KillianGricourt) [inline Parser/jdl_parser.py:8]: Commented code shouldn't stay here

File: SQL/Config_BDD.py [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+        conn.commit()
+        conn.execute(clusterDahu)
+        conn.commit()
+
+    """
+    stmt = insert(campagnes).values(grid_user="clément", state=campaign_state(2), submission_time=datetime.now(), type="test")
+
+    with engine.connect() as conn:
+        result = conn.execute(stmt)
+        #conn.execute(insert(cluster_table).values(name="Luke", api_url="/prout", api_username="dova", api_password="zer", batch=api(1), api_auth_type="pouet"))
```
Comment (KillianGricourt) [inline SQL/Config_BDD.py:333]: ???

File: SQL/queries.py [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+
+def selectBagOfTaskCampaign (id):
+    selectBagOfTaskCampaign = select(bag_of_tasks.c.campaign_id).where(bag_of_tasks.c.id == id)
+    return executeQueryReturn (selectBagOfTaskCampaign)
+
+def selectBagOfTaskParam (id):
+    selectBagOfTaskParam = select(bag_of_tasks.c.param_id).where(bag_of_tasks.c.id == id)
+    return executeQueryReturn (selectBagOfTaskParam)
+
+"""
```
Comment (KillianGricourt) [inline SQL/queries.py:336]: idem

File: Test/TestJdlParser.py [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+    def test_getClusters(self):
+        # import jdl
+        jdl_file = prefix + "testNoCluster.jdl"
+        jdl = get_json(jdl_file)
+
+        self.assertEqual(getClusters(jdl), [])
+
+
+## ONE CLUSTER USED + ONE PARAMETER + JDL FILE
+#class OneParameterJdlUT(TestManager):
```
Comment (KillianGricourt) [inline Test/TestJdlParser.py:73]: ?

File: Test/TestJdlParser.py [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+
+    def test_getClusters(self):
+        jdl_file = prefix + "test1.json"
+        jdl = get_json(jdl_file)
+
+        self.assertEqual(getClusters(jdl), ["luke"])
+
+
+## SEVERAL CLUSTER USED + SEVERAL PARAMETERS + JSON FILE
+#class SeveralParameterJdlUT(TestManager):
```
Comment (KillianGricourt) [inline Test/TestJdlParser.py:167]: ?

**Review by KillianGricourt — CHANGES_REQUESTED** [review]

**Review by KillianGricourt — DISMISSED** [review]
Ok

**Review by DiegoCDO — APPROVED** [review]

**Pull request merged**